sparkle/gui/controlwindow.py

```diff
@@ -279,6 +281,8 @@ def saveInputs(self, fname):
         savedict['advanced_options'] = self.advanced_options
         savedict['stim_view_defaults'] = StimulusView.getDefaults()
 
+        savedict['tuning_curve'] = TCFactory.defaultInputs.copy() 
```

Comment:
It's not actually necessary to get a copy here. It won't hurt, and the efficiency gain will be negligible, but for simplicity's sake, I would take it out.